modules/ROOT/pages/addition/agent-installation/agent-config-file.adoc

```diff
@@ -1,6 +1,6 @@
 = Agent configuration file
 
-The agent configuration file is used by a self-registering agent to persist its initial configuration, which is required for restarts.
+The agent configuration file is used by a self-registering agent to persist its token information, which is required for restarts.
```

Comment:
the reason to name it 'initial configuration' was that it's not just token information for use cases such as Daemonsets in K8s envs, it's possible to provide all agent config through this file

Reply:
True. But this wording doesn't say it can't be used persist other stuff, just that it needs the file to persist the token info, which should be the most important use for it.

Reply:
true for current iteration of self-registering but remote configuration implementation would enable persisting additional configuration items too. From agent's perspective, on every restart this file would be it's initial configuration, hence the name. I agree that this change makes in the current release though.

Reply:
Hmm - ok,  I found 'initial configuration' really confusing - for me the 'initial configuration' is whatever the user has supplied. 

Maybe 'the configuration the self-registering agent receives from the server'